src/AppInstallerCLICore/Argument.h

```diff
@@ -14,6 +14,8 @@
 
 #define APPINSTALLER_CLI_ARGUMENT_IDENTIFIER_CHAR       '-'
 #define APPINSTALLER_CLI_ARGUMENT_IDENTIFIER_STRING     "-"
+#define APPINSTALLER_CLI_LONG_ALIAS_IDENTIFIER_CHAR     '+'

```

Comment:
Not a fan of adding a new syntax here; would prefer the alternate name strategy.